tests/providers/amazon/aws/secrets/test_secrets_manager.py

```diff
@@ -52,14 +52,14 @@ def test_get_conn_value_full_url_mode(self):
         assert "postgresql://airflow:airflow@host:5432/airflow" == returned_uri
 
     @pytest.mark.parametrize(
-        "full_url_mode, login, host",
+        "are_secret_values_urlencoded, login, host",
         [
-            (False, "is url encoded", "not%20idempotent"),
-            (True, "is%20url%20encoded", "not%2520idempotent"),
+            (True, "is url encoded", "not%20idempotent"),
+            (False, "is%20url%20encoded", "not%2520idempotent"),
```

Comment:
This seems backwards?

Reply:
The test is basically saying:

- If the values are URL-encoded (True), then the expected value will have the escaping removed `"is%20url%20encoded" -> "is url encoded"`
- If the values are _not_ URL-encoded (False), then the expected value will not have the escaping removed `"is%20url%20encoded" -> "is%20url%20encoded"`

Admittedly this is very confusing, not least of all because I am flipping the kwarg `full_url_mode` to `are_secret_values_urlencoded`.

This was a really weirdly specified test, looking back at it.

Reply:
This one is nice - @o-nikolas - what's your take ?